actool/manifest.go

```diff
@@ -250,6 +264,23 @@ func patchManifest(im *schema.ImageManifest) error {
 		}
 	}
 
+	if patchSeccompMode != "" {
```

Comment:
I'm ok with saying `set MUST NOT be empty`. It's easier to change that in that direction if we ever need to allow empty set later.